lib/README.md

```diff
@@ -0,0 +1,169 @@
+[![GitHub tag (latest SemVer)](https://img.shields.io/github/tag/sifive/wit.svg?label=release)](https://github.com/sifive/wit/releases/latest)
+[![Build Status](https://travis-ci.com/sifive/wit.svg?branch=master)](https://travis-ci.com/sifive/wit)
+[![Github](https://img.shields.io/github/license/sifive/wit.svg?color=blue&style=flat-square)](LICENSE)
+
+# Wit
+Workspace Integration Tool
+
+## What is this?
+Wit is a tool for managing workspaces. It is intended as a supplement to (not a replacement for) Git-based workflows.
+The goal of wit is to enable development in a multiple user and repository environment.
+
+A wit workspace is composed of one or more packages. A package is a git repository.
+Each package may optionally contain a wit-manifest.json file which defines other packages upon which it depends.
+Wit resolves this hierarchy of dependencies and generates a flattened directory structure in which each package
+may exist only once.
+
+## How does Wit deduplicate packages?
+When multiple versions of the same package are requested, Wit chooses the latest requested version, making sure the selected version's commit is a descendant of every other requested versions' commits.
+
+## Getting started
+
+The best way to learn wit is to check out the [tutorial](share/doc/wit/tutorial.md).
```

Comment:
Note that every single one of these relative URLs is no longer correct, at least from the perspective of the `lib/README.md` location.

I'm not sure we can really get away with using the same README for both GitHub and the Python package, given that the README is written from the perspective of the source repository, not of the Python package itself. I know it's a copout, but if you must have a README in the Python package, can you just create a separate document that  links to the GitHub repository?